### PR TITLE
CDPS-325 - improve contrast for alert and case note details

### PR DESCRIPTION
**Pull request by jamesgrant-moj**

### CODE REVIEW

File: assets/scss/pages/_alerts-page.scss [hunk 1]
```diff
@@ -56,7 +56,7 @@
   justify-content: space-between;
   flex-basis: min-content;
   padding-bottom: 30px;
-  border-bottom: 1px solid govuk-colour('dark-grey');
+  border-bottom: 1px solid govuk-colour('black');
```
Comment (davidmiddletonmoj) [inline assets/scss/pages/_alerts-page.scss:59]: This is the wrong rule - need to change the font colour for the dates. It's in the `&__metadata` rule
```
  &__metadata {
    font-size: 16px;
    font-weight: 300;
    color: $govuk-secondary-text-colour;
  }
```

File: assets/scss/pages/_case-notes-page.scss [hunk 2]
```diff
@@ -58,7 +58,7 @@
   gap: 20px;
 
   padding-bottom: 30px;
-  border-bottom: 1px solid govuk-colour('dark-grey');
+  border-bottom: 1px solid govuk-colour('black');
```
Comment (davidmiddletonmoj) [inline assets/scss/pages/_case-notes-page.scss:61]: Same as previous comment

**Comment by jamesgrant-moj** [comment]
Thanks @davidmiddletonmoj  - ready for you to review.

**Review by davidmiddletonmoj — APPROVED** [review]
LGTM

**Pull request merged**